globals.sh

```diff
@@ -28,6 +28,8 @@ MIXER_OPTS=${MIXER_OPTS:-""}
 NUM_DELTA_BUILDS=${NUM_DELTA_BUILDS:-10}
 # Supported Mixer Version
 MIXER_MAJOR_VER=${MIXER_MAJOR_VER:-6}
+# Width of MCA statistics table, 0 uses terminal width.
+MCA_TABLE_WIDTH=${MCA_TABLE_WIDTH:-0}
```

Comment:
Most distro-factory usage  is via automation with may imply that there is no terminal width. So probably we want a better default like either 80 or 120 (preferable) than 0 on distro-factory.